requirements/common.in

```diff
@@ -52,7 +52,12 @@ django-auth-ldap
 django-bitfield
 
 # Needed for Android push notifications
-python-gcm
+# grpcio-status needs to be pinned to 1.62.* because higher
+# versions demand protobuf>5.0, which conflicts with various
+# other big dependencies of firebase-admin. pip fails to
+# find this resolution by itself.
+grpcio-status==1.62.*
```

Comment:
Is there an upstream issue for this?

Reply:
Not sure, didn't find this issue in `firebase-admin-python`. The closest was https://github.com/firebase/firebase-admin-python/issues/585 also involving protobuf, but aside of that similarity it's pretty old and somewhat different in the end.

And locally I was able to `pip3 install firebase-admin` on my own system, which grabbed `grpcio-status` version `1.62.2` correctly by itself. So I don't know if it's something more specific to our dev env or `update-locked-requirements` that made the issue manifest.